android-patches/scripts/CopyToRepo.bat

```diff
@@ -1,27 +1,27 @@
-@if "%DEBUG%" == "" @echo off

```

Comment:
I actually looked into this and there have been several commits upstream to keep the CRLF line endings on window bat scripts, but otherwise match the OS elsewhere. I'm not sure what like endings these are but hopefully fine 🤷🏾